### PR TITLE
Updating cli docs for `--browser` flag behavior

### PR DESCRIPTION
**Pull request by raygdev**
closes [#22003](https://github.com/cypress-io/cypress/issues/22003)

Updates the behavior of the `--browser` flag when passed alone to the cli.

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
@raygdev These changes look a bit off - did you pull down the 13.14.0 branch first?

**Comment by raygdev** [comment]
@jennifer-shehane I thought I did. I forked the repo, set the upstream remote to cypress-documentation and ran `git pull upstream v13.14.0`. Did I pull from the wrong branch?

**Comment by raygdev** [comment]
Maybe it's because I didn't pull the changes in from the PR that you had already opened?

File: docs/guides/references/changelog.mdx [hunk 1]
```diff
@@ -18,6 +18,7 @@ _Released 8/27/2024_
 - `.type({upArrow})` and `.type({downArrow})` now also works for date, month, week, time, datetime-local and range input types. Addresses [#29665](https://github.com/cypress-io/cypress/issues/29665).
 - Added a `CYPRESS_SKIP_VERIFY` flag to enable suppressing Cypress verification checks. Addresses [#22243](https://github.com/cypress-io/cypress/issues/22243).
 - Updated the protocol to allow making Cloud API requests. Addressed in [#30066](https://github.com/cypress-io/cypress/pull/30066).
+- Passing `--browser` flag alone will automatically launch browser after being guided through project and/or testing type selection Addressed in [#28538](https://github.com/cypress-io/cypress/pull/28538)
```
Comment (MikeMcC399) [inline docs/guides/references/changelog.mdx:21]:
```suggestion
- Passing `--browser` flag alone will automatically launch browser after being guided through project and/or testing type selection. Addressed in [#28538](https://github.com/cypress-io/cypress/pull/28538).
```

**Review by jennifer-shehane — APPROVED** [review]

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**